mysql-test/suite/connection_control/inc/set_after_marker.inc

```diff
@@ -0,0 +1,10 @@
+# set after marker
+# $SERVER_RESPONSE_TIME [IN] Expected delay
+disable_query_log;
+disable_result_log;
+SET @after= TIMESTAMP(current_time());
+SET @server_response_time= TIMESTAMPDIFF(SECOND, @before, @after);
```

Comment:
No, we absolutely cannot have time-based tests in the test suite. There were many attempts of doing that, and they all ended the same way — the test being disabled or removed. There's no need to try to walk down this road again.